modules/serverless-logic-modifying-managed-properties-workflow-services.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-configuring-workflow-services
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-modifying-managed-properties-workflow-services_{context}"]
+= Managed properties in workflow services
+
+The {ServerlessLogicOperatorName} uses managed properties to control essential runtime behavior. These values are stored separately and override user-defined properties during each reconciliation cycle. You can also apply custom managed properties globally by updating the `SonataFlowPlatform` resource within the same namespace.
+
+Some properties used by the {ServerlessLogicOperatorName} are managed properties and cannot be changed through the standard user configuration. These properties are stored in a dedicated `ConfigMap`, typically named `<workflow-name>-managed-props`. If you try to modify any managed property directly, the Operator will automatically revert it to its default value, but it will preserve your other user-defined changes.
+
+[NOTE]
+====
+You cannot override the default managed properties set by the Operator using global managed properties. These defaults are always enforced during reconciliation.
+====
+
+The following table lists some core managed properties:
+
+.Managed properties overview
+[cols="2,1,1",options="header"]
+|====
+|Property Key 
+|Immutable Value
+|Profile
+
+|`quarkus.http.port`
+|`8080`
+|`all`
+
+|`quarkus.http.host`
+|`0.0.0.0`
+|`all`
+
+|`org.kie.kogito.addons.knative.eventing.health-enabled`
+|`false`
+|`dev`
+
+|====
```

Comment:
@wmedvede is that all?

Reply:
@ricardozanini it's not all, but to add all, considering all the combinations persistence yes/no, persistence yes/no, DI yes/no, JS yes/no, direct http, knative eventing, etc,  will lead the into something very dificult to understand. I'll add some more of them to make it more realistic, but to include all is impractical.